src/track/TileLoader.ts

```diff
@@ -38,12 +38,25 @@ export class TileLoader<TilePayload, BlockPayload> {
         x1: number,
         samplingDensity: number,
         loadEmptyTiles: boolean,
-        callback: (tile: Tile<TilePayload>) => void
+        callback: (tile: Tile<TilePayload>) => void,
+        errorLoadingCallback?: () => void
     ) {
         let lodLevelFractional = Scalar.log2(Math.max(samplingDensity, 1));
         let lodLevel = Math.floor(lodLevelFractional);
 
-        return this.forEachTileAtLod(x0, x1, lodLevel, loadEmptyTiles, callback);
+        if (!errorLoadingCallback) {
+            errorLoadingCallback = () => {
+                const message: string = 'Error reading chromosome';
+                const element = document.querySelector<HTMLElement>('.valis-error-message'); 
+        
+                if (element) {
```

Comment:
I know this works but I don't quite see how - won't this always be true?

Reply:
Defensive programming. The code does not own the markup so it should assume it may not exist. I can take it out if you strongly dislike it.

Reply:
No, no, this is great. I should probably do more defensive programming. I just thought that it was there because sometimes element might not exist but couldn't see when that would be true.